spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -164,7 +167,27 @@
         expect(JSON.parse(response.body)['video_link']).to eq(new_video_link)
       end
     end
-
+    
+    context 'with valid level' do
+      let(:recipe_put_level) do
+        {
+          id: recipe.id,
+          recipe: { level: new_level }
+        }
+      end 
+      
+      it 'updated level' do
+        expect do
+          put :update, params: recipe_put_level
```

Comment:
You can extract ``` put :update, params: recipe_put_level ``` to let, something like
```let(:update_request) { put :update, params: recipe_put_level }```

and call it with expect{ update_request }.to ...